pkg/machine/applehv/machine.go

```diff
@@ -583,6 +619,29 @@ func (m *MacMachine) Stop(name string, opts machine.StopOptions) error {
 	if vmState != machine.Running {
 		return machine.ErrWrongState
 	}
+
+	defer func() {
+		// I think "soft" errors here is OK
+		gvPid, err := m.GvProxyPid.Read()
+		if err != nil {
+			logrus.Error(err)
```

Comment:
Please add some context to the errors via `Errorf("Reading gvproxy PID: %v", err)` etc.  That will help identifying where things break.  Sometimes the returned errors are super generic and hard to debug (see the `exit 255` fart in qemu).